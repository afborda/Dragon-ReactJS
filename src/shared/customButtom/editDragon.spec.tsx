import { render } from '@testing-library/react'
import React from 'react'
import CustomButtom from '.'

describe('<ListDragon/>', () => {
  it('should  render the heading', () => {
    const { container } = render(<CustomButtom />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
