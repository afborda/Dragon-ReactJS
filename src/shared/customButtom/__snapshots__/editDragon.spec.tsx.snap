// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListDragon/> should  render the heading 1`] = `
<div>
  <button
    class="style__Buttom-gGydxe kWrCVF"
  />
</div>
`
